Makefile

```diff
@@ -58,12 +57,18 @@ build: ## build potsie development image
 	@$(COMPOSE) build app
 .PHONY: build
 
+clean: ## remove generated and downloaded files
+	rm -rf ./var
+.PHONY: clean
+
 compile: \
 	tree \
 	$(targets)
 compile: ## compile jsonnet sources to json
 .PHONY: compile
 
+down: \
+	clean
```

Comment:
I would have done the opposite (`down` as a `clean` dependency) as `clean` is a more popular / expected rule for a fresh start. WDYT?

Reply:
Sounds good) Thanks) updated this part)